c2corg_common/fields_report.py

```diff
@@ -0,0 +1,67 @@
+DEFAULT_FIELDS = [
+    'locales',
+    'locales.title',
+    'locales.summary',
+    'locales.description',
+    'locales.place',
+    'locales.route_study',
+    'locales.conditions',
+    'locales.training',
+    'locales.motivations',
+    'locales.group_management',
+    'locales.risk',
+    'locales.time_management',
+    'locales.safety',
+    'locales.reduce_impact',
+    'locales.modifications',
+    'locales.other_comments',
+    'geometry',
+    'geometry.geom',
+    'lon',
+    'lat',
```

Comment:
I don't think we need ``lon`` and ``lat`` since the coordinates are actually passed through ``geometry.geom``. As for waypoints: https://github.com/c2corg/v6_common/blob/master/c2corg_common/fields_waypoint.py